testapp/settings.py

```diff
@@ -0,0 +1,61 @@
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "testapp",
+    "phac_aspc.django.helpers",
+]
+USE_I18N = (True,)
+LANGUAGE = (
+    ("fr-ca", "French"),
+    ("en-ca", "English"),
+)
+LANGUAGE_CODE = "en-ca"
+MIDDLEWARE = [
+    "django.middleware.security.SecurityMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+]
+
+DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+AUTH_USER_MODEL = "testapp.User"
+
+TEST_RUNNER = "testapp.tests.pytest_runner.PytestRunner"
```

Comment:
This is just to allow running tests using `manage.py test`, I find it useful, there's some different CLI options too